browser/extensions/api/brave_rewards_api.cc

```diff
@@ -143,6 +145,8 @@ void BraveRewardsTipTwitterUserFunction::OnTwitterPublisherInfoSaved(
 
   auto params_dict = std::make_unique<base::DictionaryValue>();
   params_dict->SetString("publisherKey", publisher_info->id);
+  params_dict->SetString(
+      "url", contents ? contents->GetLastCommittedURL().spec() : std::string());
```

Comment:
@NejcZdovc Wasn't sure if there was a better approach here... Perhaps I should have retrieved the screen name from the database, but then we'd have to expose that function in the rewards service. Wasn't sure if it was worth it, but let me know what you think. Maybe I'm missing something more obvious here.

Reply:
you have access to `publisher_info` object here which has url inside, so you could just do `publisher_info->url`

Reply:
Good point, fixed!